src/main/java/seedu/taassist/model/ModelManager.java

```diff
@@ -255,4 +274,18 @@ public ModuleClass getFocusedClass() {
     public SimpleStringProperty getFocusLabelProperty() {
         return focusLabelProperty;
     }
+
+    //=========== Handles queried session state ==================================================================
+    @Override
+    public void querySessionData(Session targetSession) {
+        requireAllNonNull(focusedClass, targetSession);
+        // Should only be called when Model is in focus mode and focused class definitely contains the session.
+        assert(focusedClass.hasSession(targetSession));
```

Comment:
Java assertions do not require brackets. 
```suggestion
        assert focusedClass.hasSession(targetSession);
```